app/views/articles/_actions.html.erb

```diff
@@ -19,7 +19,7 @@
       <%= render partial: "articles/reaction_button",
                  locals: {
                    category: :readinglist,
-                   description: "Saved",
+                   description: "Reading list",
```

Comment:
as it could be either `saved` or `unsave`, why keep `saved` ? However, no problem with that change too.

Reply:
It's a fair point - the description is being used for the text that appears on hover over. In the future it might be worth refactoring to also dynamically change between save/unsave etc, but it doesn't have as strong an impact on usability (sighted users who are seeing that text can already see the state of the button and whether it's in a saved/unsaved state or not).

Maybe something we can consider in a separate PR in future, and keep the scope of this one more contained 👍